src/main/java/step2/winning/Winning.java

```diff
@@ -0,0 +1,31 @@
+package step2.winning;
+
+import java.util.Arrays;
+
+public enum Winning {
+    FIRST(6, 2000000000, false),
```

Comment:
(간단한 가독성으로)

```suggestion
    FIRST(6, 2_000_000_000, false),
```
이런식으로 쓴다면 우리가 기본적으로 숫자를 쓸때 `,`를 쓰는것처럼 가독성이 좋아질수 있습니다.